components/identity/create-identity.html

```diff
@@ -167,27 +167,16 @@ <h1 class="headline">Create an Identity</h1>
           </div>
         </div>
       </fieldset>
-
       <br />
-
-      <fieldset>
-        <legend>Service Agreement</legend>
-
-        <div class="form-group" br-property-path="agreementChecked">
-          <div class="col-md-offset-3 col-md-7">
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="model.agreementChecked"> I agree to the <a target="_blank" href="/legal#tos">Terms of Service</a> and <a target="_blank" href="/legal#pp">Privacy Policy</a>.
-              </label>
-            </div>
-          </div>
-        </div>
-      </fieldset>
-
+      <br-agreement
+        ng-model="model.agreementAccepted"
+        br-agreement-group="brIdpJoinForm">
```

Comment:
Would prefer the module name spelled out, I think... and probably better to leave "form" out of the title, eg: `bedrock-idp.join`. So that title would be for the group of agreements required to "join" per the "bedrock-idp" module.